api/v1beta1/openstackdataplanenodeset_types.go

```diff
@@ -52,6 +52,7 @@ type OpenStackDataPlaneNodeSetSpec struct {
 	Env []corev1.EnvVar `json:"env,omitempty"`
 
 	// +kubebuilder:validation:Optional
```

Comment:
you have to drop this
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_dataplane-operator/591/pull-ci-openstack-k8s-operators-dataplane-operator-main-precommit-check/1742579977779417088

Reply:
I actually dropped the default value. I think that's the right thing to do here.